src/app/main/component/eco-news/services/eco-news-comments.service.ts

```diff
@@ -15,9 +15,16 @@ export class EcoNewsCommentsService implements CommentsService {
 
   constructor(private http: HttpClient) {}
 
-  addComment(ecoNewsId: number, text: string, id = 0): Observable<AddedCommentDTO> {
-    const body = { parentCommentId: id, text };
-    return this.http.post<EcoNewsAddedCommentDTO>(`${this.backEnd}eco-news/${ecoNewsId}/comments`, body);
+  addComment(ecoNewsId: number, text: string, id = 0): Observable<EcoNewsAddedCommentDTO> {
```

Comment:
should not change the return type, as the implementation should be identical to the abstract method, ensuring consistency of argument names and return types.

use plz, `AddedCommentDTO`